pkg/steps/multi_stage/gen.go

```diff
@@ -178,9 +183,26 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 		if step.RunAsScript != nil && *step.RunAsScript {
 			addCommandScript(commandConfigMapForTest(s.name), pod)
 		}
+		if s.vpnImage != "" {
+			caps := coreapi.Capabilities{
+				Add:  []coreapi.Capability{"NET_ADMIN"},
+				Drop: []coreapi.Capability{"ALL"},
+			}
+			seLinuxOpts := coreapi.SELinuxOptions{
+				User: "system_u",
+				Role: "system_r",
+				// TODO create a more restricted SELinux context
+				// This one happens to be in every cluster and have the
+				// permission to use /dev/net/tun and configure networking, but
+				// has *many* more permissions than are required here.
+				Type:  "container_runtime_t",
```

Comment:
I intend to address this in a later PR.

The VPN container is completely under control of `ci-operator` and the cluster profile owner and has no influence from the test container, so this lax set of permissions is bad, but not catastrophic.

Reply:
https://issues.redhat.com/browse/DPTP-2850